.travis.yml

```diff
@@ -8,11 +8,12 @@ matrix:
     - python: 3.6
     - python: 3.4.1
 
-script:  # Handles creating a blank config file so imports dont fail
-  - mkdir -p instance
-  - touch instance/config.py
+env:
+  - APP_CONFIG_NAME=test
+
+script:
   - pytest
-  - flake8 middleware/ --ignore=F403,F405
+  - flake8 middleware/ --ignore=F401,F403,F405
```

Comment:
No, it only flags modules that are imported and not used. 403 and 405 are to do with * imports.